src/Bicep.Core.Samples/Files/NestedResources_LF/main.json

```diff
@@ -26,10 +26,10 @@
       "name": "[format('{0}/{1}/{2}', 'basicParent', 'basicChild', 'basicGrandchild')]",
       "properties": {
         "size": "[reference(resourceId('My.Rp/parentType', 'basicParent')).size]",
-        "style": "[reference(resourceId('My.Rp/parentType/childType', split(format('{0}/{1}', 'basicParent', 'basicChild'), '/')[0], split(format('{0}/{1}', 'basicParent', 'basicChild'), '/')[1])).style]"
+        "style": "[reference(resourceId('My.Rp/parentType/childType', 'basicParent', 'basicChild')).style]"
       },
       "dependsOn": [
-        "[resourceId('My.Rp/parentType/childType', split(format('{0}/{1}', 'basicParent', 'basicChild'), '/')[0], split(format('{0}/{1}', 'basicParent', 'basicChild'), '/')[1])]",
+        "[resourceId('My.Rp/parentType/childType', 'basicParent', 'basicChild')]",
```

Comment:
>basicParent [](start = 52, length = 11)

This was annoying me when I was converting some of our internal templates to Bicep.